definitions/divorce/json/CaseEvent/CaseEvent.json

```diff
@@ -360,10 +360,22 @@
     "Name": "DA Granted",
     "Description": "DA Granted",
     "DisplayOrder": 1,
-    "PreConditionState(s)": "AwaitingDecreeAbsolute",
+    "PreConditionState(s)": "AwaitingDecreeAbsolute;DAPending",
```

Comment:
should this go live as soon as possible? no release?

Reply:
indeed. Config only change.